Libraries/LibMedia/PlaybackManager.h

```diff
@@ -79,6 +76,11 @@ class MEDIA_API PlaybackManager final : public AtomicRefCounted<PlaybackManager>
 
     Function<void()> on_playback_state_change;
     Function<void(DecoderError&&)> on_error;
+    Function<void(AK::Duration)> on_metadata_parsed;
+    Function<void(DecoderError&&)> on_unsupported_format_error;
+    Function<void(TrackType, Track const&)> on_track_added;
```

Comment:
We could probably move these three added callbacks first, since they should normally be called before the existing two.